components/ListView/ListItemImages.js

```diff
@@ -63,173 +129,233 @@ class ListItemImages extends React.Component {
+      case "FINISHED":
+        actions = (
+          <>
+            <li>
+              <a href={this.props.downloadUrl} role="button" download>
+                <FormattedMessage defaultMessage="Download" />
+              </a>
+            </li>
+            <li key="delete">
+              <a href="#" role="button" onClick={e => this.handleShowModalDeleteImage(e)}>
+                <FormattedMessage defaultMessage="Delete" />
+              </a>
+            </li>
+          </>
+        );
+        break;
+      case "RUNNING":
+        actions = (
+          <>
+            <li>
+              <a href="#" role="button" onClick={e => this.handleShowModalStop(e)}>
+                <FormattedMessage defaultMessage="Stop" />
+              </a>
+            </li>
+          </>
+        );
+        break;
+      case "WAITING":
+        actions = (
+          <>
+            <li>
+              <a href="#" role="button" onClick={e => this.handleCancel(e)}>
+                <FormattedMessage defaultMessage="Stop" />
+              </a>
+            </li>
+          </>
+        );
+        break;
+      case "FAILED":
+        actions = (
+          <>
+            <li>
+              <a href="#" role="button" onClick={e => this.handleDelete(e)}>
+                <FormattedMessage defaultMessage="Remove" />
+              </a>
+            </li>
+          </>
+        );
+        break;
+      default:
+        actions = undefined;
+        break;
+    }
+    const noLogs = listItem.queue_status === "WAITING" || listItem.queue_status === "STOPPING";
+    const logsButton = (
+      <div
+        aria-hidden={noLogs}
+        className={`pf-c-data-list__item-action ${noLogs ? "cc-u-not-visible" : ""}`}
+      >
+        <Button
+          variant={`${this.state.logsExpanded ? "primary" : "secondary"}`}
+          aria-expanded={this.state.logsExpanded}
+          aria-controls={`${listItem.id}-logs`}
+          onClick={this.handleLogsShow}
+        >
+          {formatMessage(messages.imageLogs)}
+        </Button>
+      </div>
     );
-    let uploadsButton;
-    if (listItem.uploads !== undefined && listItem.uploads.length > 0) {
-      uploadsButton = (
-        <button className="btn btn-link" onClick={this.handleUploadsShow} type="button">
-          {this.state.uploadsExpanded && <FormattedMessage defaultMessage="Hide uploads" />}
-          {!this.state.uploadsExpanded && <FormattedMessage defaultMessage="Show uploads" />}
-        </button>
+    let uploadsToggle;
+    if (listItem.uploads !== undefined) {
+      uploadsToggle = (
+        <DataListToggle
+            onClick={this.handleUploadsShow}
+            isExpanded={this.state.uploadsExpanded}
+            id="uploads-toggle"
+            aria-label={`${formatMessage(messages.imageUploads)} ${this.props.blueprint}-${listItem.version}-${listItem.compose_type}`}
+            aria-controls={`${listItem.id}-uploads`}
+            // ^ need to fix this attribute value
```

Comment:
What needs to be fixed here?

Reply:
Oh, I need to remove that comment. I fixed the attribute (i.e. it references a unique ID of the section in controls), but forgot to remove the comment.